### PR TITLE
Drop `toggle-files-button` feature

### LINKED ISSUE #3580: Hide "Readme" text from repositories
Does anyone have any idea of why this link exists on repo roots like https://github.com/sindresorhus/refined-github?

It points to the same page. Should we hide it? It can just be part of `refined-github.css`:

```css
.BorderGrid-cell [href="#readme"] {
	display: none;
}
```

### PR DESCRIPTION
**Pull request by roottool**
1. LINKED ISSUES:
   Closes #3580

2. TEST URLS:
   [Repository page](https://github.com/roottool/refined-github)

3. SCREENSHOT:
   **Before**
   before

   **After**
   After

   **GIF**
   Drops `toggle-files-button` feature.

### CODE REVIEW

**Comment by yakov116** [comment]
Should also go into `Previously part of Refined GitHub` with [this](https://github.blog/changelog/2020-06-23-design-updates-to-repositories-and-github-ui/) as the link

**Comment by roottool** [comment]
@yakov116 Thank you for reviewing my PR. I added the new item to `Previously part of Refined GitHub`.

**Review by fregante — APPROVED** [review]
💯

File: readme.md [hunk 1]
```diff
@@ -386,6 +385,7 @@ GitHub implemented dozens of features that used to be part of Refined GitHub 
 - [Blog post](https://github.blog/changelog/2019-12-19-improved-attribution-when-squashing-commits/): [Adds `co-authored-by` to the commit when merging PRs with multiple committers.](https://user-images.githubusercontent.com/1402241/51468821-71a42100-1da2-11e9-86aa-fc2a6a29da84.png)
 - [Blog post](https://github.com/sindresorhus/refined-github/pull/3015): [Makes it easier to tell apart commits added to the current PR versus plain commits that reference the PR.](https://user-images.githubusercontent.com/1402241/64478939-398b0a80-d1da-11e9-8c6a-bb98668cb78c.gif)
 - [Blog post](https://github.blog/changelog/2020-06-23-design-updates-to-repositories-and-github-ui/): [Widens the discussion search box.](https://user-images.githubusercontent.com/1402241/55069759-bceaf080-50bf-11e9-84d0-7707de2eb9e9.png)
+- [Blog post](https://github.blog/changelog/2020-06-23-design-updates-to-repositories-and-github-ui/): [Drops `toggle-files-button` feature.](https://user-images.githubusercontent.com/11808736/93828192-c7327e80-fca5-11ea-9557-b2c222748689.gif)
```
Comment (fregante) [inline readme.md:388]:
```suggestion
- [Blog post](https://github.blog/changelog/2020-06-23-design-updates-to-repositories-and-github-ui/): [Adds a button to toggle the repo file list.](https://user-images.githubusercontent.com/11808736/93828192-c7327e80-fca5-11ea-9557-b2c222748689.gif)
```

File: readme.md [hunk 2]
```diff
@@ -386,6 +385,7 @@ GitHub implemented dozens of features that used to be part of Refined GitHub 
 - [Blog post](https://github.blog/changelog/2019-12-19-improved-attribution-when-squashing-commits/): [Adds `co-authored-by` to the commit when merging PRs with multiple committers.](https://user-images.githubusercontent.com/1402241/51468821-71a42100-1da2-11e9-86aa-fc2a6a29da84.png)
 - [Blog post](https://github.com/sindresorhus/refined-github/pull/3015): [Makes it easier to tell apart commits added to the current PR versus plain commits that reference the PR.](https://user-images.githubusercontent.com/1402241/64478939-398b0a80-d1da-11e9-8c6a-bb98668cb78c.gif)
 - [Blog post](https://github.blog/changelog/2020-06-23-design-updates-to-repositories-and-github-ui/): [Widens the discussion search box.](https://user-images.githubusercontent.com/1402241/55069759-bceaf080-50bf-11e9-84d0-7707de2eb9e9.png)
+- [Blog post](https://github.blog/changelog/2020-06-23-design-updates-to-repositories-and-github-ui/): [Adds a button to toggle the repo file list.](https://user-images.githubusercontent.com/11808736/93828192-c7327e80-fca5-11ea-9557-b2c222748689.gif)
```
Comment (yakov116) [inline readme.md:388]:
```suggestion
- [Blog post](https://github.blog/changelog/2020-06-23-design-updates-to-repositories-and-github-ui/): [Adds a button to toggle the repo file list.](https://user-images.githubusercontent.com/1402241/35480123-68b9af1a-043a-11e8-8934-3ead3cff8328.gif)
```

I think the old GIF was better

**Review by yakov116 — APPROVED** [review]

**Comment by fregante** [comment]
Thanks for the PR @roottool, unfortunately we're probably not doing to drop the feature after all, because

> I agree, the reason we wanted to drop the feature was due to the Readme link already existing. However even that, **only exists on the root of the repository.**

https://github.com/sindresorhus/refined-github/issues/3588#issuecomment-696695805

**Pull request closed without merging**